src/components/calcite-input/calcite-input.tsx

```diff
@@ -47,7 +48,16 @@ export class CalciteInput {
   @Prop({ mutable: true, reflect: true }) alignment: "start" | "end" = "start";
 
   /** input value */
-  @Prop({ mutable: true }) value?: string | null = "";
+  @Prop({ mutable: true, reflect: true }) value?: string = "";
+
+  /** input step */
+  @Prop({ mutable: true, reflect: true }) step?: string = "";
```

Comment:
do these actually need to be mutable? Are you changing them from inside this component?